shared/components/ember-output.jsx

```diff
@@ -0,0 +1,73 @@
+/* global React */
+
+export default React.createClass({
+    displayName: 'EmberOutput',
+
+    propTypes: {
+        source  : React.PropTypes.string.isRequired,
+        context : React.PropTypes.object.isRequired,
+        onChange: React.PropTypes.func
+    },
+
+    componentWillReceiveProps: function (nextProps) {
+        if (nextProps.source !== this.props.source) {
+            this.setState({});
+        }
+
+        Object.keys(nextProps).forEach(function (key) {
+            if (key === 'locales') {
+                Ember.set(this.service, 'locales', nextProps[key]);
+            }
+
+            this.controller.set(key, nextProps[key]);
+        }, this);
+    },
+
+    localesChanged: function () {
+        this.props.onChange(this.service.get('locales'));
+    },
+
+    componentWillUnmount: function () {
+        this.service.off('localesChanged', this, this.localesChanged);
+    },
+
+    componentDidMount: function () {
+        window.Ember.Application.initializer({
+            name : this.props.exampleId,
+            after: 'ember-intl-standalone',
+
+            initialize: function (container, app) {
+                var domElement = this.refs.placeholder.getDOMNode();
+                var controller = this.controller = container.lookupFactory('controller:basic').create();
+
+                this.service = container.lookup('intl:main');
+
+                if (!container.has('view:default')) {
+                    container.register('view:default', Ember.View);
+                }
+
+                this.service.on('localesChanged', this, this.localesChanged);
```

Comment:
Anytime you call intl.set('locales', ..); it triggers this action.

https://github.com/yahoo/formatjs-site/pull/166/files#diff-7dbab17105c7b77d9dab6158728b2cd6R19

Is where it's being triggered in this example.